addons/website_sale/views/templates.xml

```diff
@@ -794,7 +863,10 @@
             <t t-set="entries" t-value="categories"/>
         </t>
 
-        <div t-if="entries" class="o_wsale_filmstip_container d-flex align-items-stretch mb-2 overflow-hidden">
+        <div
+            t-if="entries"
+            class="o_wsale_filmstip_container d-flex align-items-stretch overflow-hidden"
+        >
```

Comment:
I know that it's the expected behaviour according to the design @paju-odoo validated, but I'm still not sure about the visual result, sorry 😕 
I see at least 3 issues:
- By default, the "fullscreen" tiny scrollbar resembles too much an horizontal separator. The effect it's even more prominent if your OS's scrollbar design doesn’t match.

- If a sidebar is available (normally it's the case) I'd like to visually prioritize that rather than the filmstrip. The upper left corner it's way too important since it's the first zone that the user will "scan" with his eyes.

|  |
|--------|
| ![image](https://github.com/user-attachments/assets/7c32188b-8b22-45ba-b132-3859dba14269) | 


- When you have just one category (or a few) the space it's not well optimised. Somehow it was the case even before, but in this PR the issue affects the entire page 😕 (see master screenshoot for comparison).

|  |
|--------|
|  this PR ![image](https://github.com/user-attachments/assets/8b2526b9-ccb9-4f27-b39f-b1920a94da75) |
|  master ![image](https://github.com/user-attachments/assets/850403d9-9a33-4933-bd16-f2821a491b8f) |
 
In conclusion I'd keep the filmistrip where it is .
I'm well aware that some users may find this layout visually interesting, but eventually it should be an option; not default IMO

Reply:
@stefanorigano 
1. is it really worse than today ? I think it was a bit the idea of moving categories above, separate ir more clearly. Can't we simply improve the layout ? 
2. Not sure user with few number of categories will use this feature. If you have a few number of categories it's probably better to add the filter on the left don't you think ?

Reply:
@stefanorigano I navigated on several eshop and the few cases I found with a filter on categories on top of the shop are always above everything.

Reply:
I agree to keep the above one instead of the in grid.

Reply:
@dalaOdoo 

> 1 is it really worse than today ? 

As said, some people would love it, but others don't. FP for example, things that "master is good / better" and I do agree with him on this. Let's not impose debatable choices, we have editor options to handle such scenario

> 2 Not sure user with few number of categories will use this feature. If you have a few number of categories it's probably better to add the filter on the left don't you think ?

Nope. You may have dozens of categories, but as soon as you "navigate" them you'll end-up in a scenario with just one sub-category (or a few)

@mano-odoo 

> ... But there is room for improvement A solution such as the following on nested categories,...

Nice, I don't see such solution in your PR 😉 ... I don't think that we can ask people to wait till 19.0 to got it fixed.

Reply:
let's move forward then

Reply:
Hello @livaios, I would say it makes more sense to keep the sort by and the viewswitcher closer to the grid because it interacts directly with it. 

Filtering or changing the view in list is closely related to the grid while the search and the filmstrip are navigation elements. They serve a different purpose. It was more clear with the borders like in the initial mockup though: 
![image](https://github.com/user-attachments/assets/138d4b33-20e6-4e39-9947-3b5979833677)


Note in the end after a whole lot of discussion, FP prefers what we have in master for the /shop page. I'll update this PR with only the minor fixes (sort by dropdown) and the navigation fixes on the /product page.